src/modules/person/persistence/person.repo.ts

```diff
@@ -58,4 +57,29 @@ export class PersonRepo {
         await this.em.persistAndFlush(person);
         return this.mapper.map(person, PersonEntity, PersonDo);
     }
+
+    public async findAll(personDo: PersonDo<false>): Promise<Option<PersonDo<true>>[]> {
```

Comment:
Option in the return Type does not make sense. The are no null or undefined entries in the array.